samples/web/asp-web-application/sample.md

```diff
@@ -64,4 +64,4 @@ When the response arrives back at `WebApp`, NServiceBus invokes the callback tha
 
 The `messageSession.Request` method takes the callback code and tells NServiceBus to invoke it when the response is received. There are several overloads of this method; the code above accepts a generic `Enum` parameter, effectively casting the return code from the server to the given enumeration type.
 
-Finally, the code updates the `Text` property of a label on the web page, setting it to the string that represents the enumeration value: sometimes `None`, sometimes `Fail`.
\ No newline at end of file
+Finally, the code updates the `Text` property of a label on the web page, setting it to the string that represents the enumeration value: sometimes `None`, sometimes `Fail`.
```

Comment:
I know you didn't change this, but why is this 'None' and 'Fail' and not 'Even' and 'Odd'?

Reply:
agreed, but there is a screenshot so not sure of the ROI to change